lib/foreman_maintain/cli/report_command.rb

```diff
@@ -0,0 +1,25 @@
+module ForemanMaintain
+  module Cli
+    class ReportCommand < Base
+      extend Concerns::Finders
+
+      option '--output', 'FILE', 'Output the generate report into FILE'
+      subcommand 'generate', 'Generates the usage reports' do
+        def execute
+          scenario = run_scenario(Scenarios::Report::Generate.new({}, [:reports])).first
+
+          # description can be used too
+          report_data = scenario.steps.map(&:data).compact.reduce(&:merge).transform_keys(&:to_s)
```

Comment:
This produces quite a lot of output, which you probably don't want in the case where you run this via cron (as otherwise you'll get a mail from cron on every invocation. You could add `--cron` or `--silent` to suppress these?